src/compiler/emitter.ts

```diff
@@ -2660,6 +2673,10 @@ module ts {
                 emit(node.whenFalse);
             }
 
+            function emitGeneratedLabel(node: GeneratedLabel) {
+                write(String(node.labelNumbers[node.label]));

```

Comment:
I feel like we tend to use the `"" + ` idiom a bit more, or maybe just `.toString()`.